### PR TITLE
POL-918 Create Master Policy Permissions List

### PR DESCRIPTION
**Pull request by nia-vf1**
### Contribution Check List

- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Comment by bryankaraffa** [comment]
Warning message is working now after fiddling with the regex:
image

**Comment by bryankaraffa** [comment]
Added error if the PT is not in the "manually_verified_policy_templates" list which is used by the generate_permissions script

image

File: cost/azure/rightsize_compute_instances/README.md [hunk 1]
```diff
@@ -64,6 +64,9 @@ For administrators [creating and managing credentials](https://docs.flexera.com/
 - [**Azure Resource Manager Credential**](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm#automationadmin_109256743_1124668) (*provider=azure_rm*) which has the following permissions:
   - `Microsoft.Compute/virtualMachines/read`
   - `Microsoft.Compute/virtualMachines/write`*
+  - `Microsoft.Compute/virtualMachines/powerOff/action` *
```
Comment (XOmniverse) [inline cost/azure/rightsize_compute_instances/README.md:67]: There should be no space between ` and *

File: cost/azure/rightsize_compute_instances/README.md [hunk 2]
```diff
@@ -64,6 +64,9 @@ For administrators [creating and managing credentials](https://docs.flexera.com/
...
+  - `Microsoft.Compute/virtualMachines/start/action` *
```
Comment (XOmniverse) [inline cost/azure/rightsize_compute_instances/README.md:68]: There should be no space between ` and *

**Review by XOmniverse — CHANGES_REQUESTED** [review]
Just a couple of minor changes

File: cost/azure/rightsize_compute_instances/README.md (hunk 1 above)
Comment (nia-vf1) [reply cost/azure/rightsize_compute_instances/README.md]: Resolved

File: cost/azure/rightsize_compute_instances/README.md (hunk 2 above)
Comment (nia-vf1) [reply cost/azure/rightsize_compute_instances/README.md]: Resolved

**Review by nia-vf1 — COMMENTED** [review]
LGTM

**Review by XOmniverse — APPROVED** [review]
LGTM

**Pull request merged**